lib/ProductOpener/Food.pm

```diff
@@ -1970,11 +1989,12 @@ sub compute_carbon_footprint_infocard($) {
 
 	if (not ((has_tag($product_ref, "countries", "en:france")) and (defined $product_ref->{ingredients_text})
 		and (length($product_ref->{ingredients_text}) > 5))) {
-		delete $product_ref->{environment_impact_level};
-		delete $product_ref->{environment_impact_level_tags};
-		delete $product_ref->{environment_infocard};
-		delete $product_ref->{environment_infocard_en};
-		delete $product_ref->{environment_infocard_fr};
+		
+		state @product_fields_to_delete = ("environment_impact_level", "environment_impact_level_tags", 
```

Comment:
I don't think we need @product_fields_to_delete to be a persistent variable, why not use my instead of state?